compiler/luci/service/src/Nodes/CirclePad.cpp

```diff
@@ -35,7 +34,8 @@ loco::TensorShape Algorithm::visit(const luci::CirclePad *node)
 {
   // TODO support non-const case
   auto paddings = loco::must_cast<luci::CircleConst *>(node->paddings());
-  return use_paddings(node, paddings);
+  auto input_shape = circle_shape(loco::must_cast<const luci::CircleNode *>(node->input()));
```

Comment:
```suggestion
  auto input = loco::must_cast<const luci::CircleNode *>(node->input());
  auto input_shape = circle_shape(input);
```

Reply:
How about
```suggestion
  auto circle_input = loco::must_cast<const luci::CircleNode *>(node->input());
  auto input_shape = circle_shape(input);
```
?
Just for more clarity.